pipeline-modules/app-service/azure/rsaf-appsec.yaml

```diff
@@ -0,0 +1,137 @@
+provisioner: azure
```

Comment:
I guess we didn't circle back on the discussion about private repo for customer specific stuff....not sure it makes sense to put a file with this name into this repo?

Reply:
i'll change the file name too, i did for description.